packages/core-mobile/app/new/common/components/LastTransactedNetworks.tsx

```diff
@@ -0,0 +1,14 @@
+import { useLastTransactedNetworks } from 'common/hooks/useLastTransactedNetworks'
+import React from 'react'
+
+/**
+ * This component is used to trigger the useLastTransactedNetworks hook.
+ * the result of the hook is saved in the MMKV storage, so we can fetch
+ * the networks as soon as the app is opened.
+ * It does not render anything.
+ * @returns {React.JSX.Element} An empty fragment.
+ */
+export const LastTransactedNetworks = (): React.JSX.Element => {
+  useLastTransactedNetworks()
+  return <></>
```

Comment:
```suggestion
  return null
```
I think the convention is null for non ui component